grape-roar.gemspec

```diff
@@ -17,5 +17,6 @@ Gem::Specification.new do |gem|
   gem.version       = Grape::Roar::VERSION
 
   gem.add_dependency 'grape'
-  gem.add_dependency 'roar', '>= 1.0'
+  gem.add_dependency 'roar', '~> 1.1.0'
+  gem.required_ruby_version = '>= 2.2.2'
```

Comment:
I figured it would make it more apparent to use a newer Ruby version, albeit you are correct. Changed.